xdsl/dialects/arm_neon.py

```diff
@@ -1,11 +1,27 @@
 from __future__ import annotations
 
+from abc import ABC
+from typing import Annotated
+
+from xdsl.backend.assembly_printer import AssemblyPrinter
+from xdsl.dialects.arm.assembly import assembly_arg_str
+from xdsl.dialects.arm.ops import ARMInstruction, ARMOperation
 from xdsl.dialects.arm.register import ARMRegisterType
-from xdsl.ir import Dialect
-from xdsl.irdl import irdl_attr_definition
+from xdsl.dialects.builtin import IntegerAttr, IntegerType, StringAttr, i8
+from xdsl.ir import Dialect, Operation, SSAValue
+from xdsl.irdl import (
+    attr_def,
+    base,
+    irdl_attr_definition,
+    irdl_op_definition,
+    operand_def,
+    result_def,
+)
 
 ARM_NEON_INDEX_BY_NAME = {f"v{i}": i for i in range(0, 32)}
 
+Imm8Attr = IntegerAttr[Annotated[IntegerType, i8]]
```

Comment:
I'm not sure we want this typealias, might as well just to `IntegerAttr[I8]` in the operation definition